src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1721,6 +1733,42 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Do not activate when Game Mode is on</value>
     <comment>"Game mode" is the Windows feature to prevent notification when playing a game.</comment>
   </data>
+  <data name="MouseUtils_MouseHighlighter.Header" xml:space="preserve">
+    <value>Mouse Highlighter</value>
+    <comment>Refers to the utility name</comment>
+  </data>
+  <data name="MouseUtils_Enable_MouseHighlighter.Header" xml:space="preserve">
+    <value>Enable Mouse Highlighter</value>
+    <comment>"Find My Mouse" is the name of the utility.</comment>
+  </data>
+  <data name="MouseUtils_Enable_MouseHighlighter.Description" xml:space="preserve">
+    <value>Customize the keyboard shortcut to enter or exit the mode to highlight mouse clicks</value>
```

Comment:
Small text improvement:
```suggestion
    <value>Customize the keyboard shortcut to enter or exit the highlighting mode</value>
```

I think we should use this description for shortcut setting and not the on/off switch. And I think we should rename `Activation shortcut` to `Shortcut` for indicating that the shortcut hase more actiins than only activating the highlight mode.

Reply:
cc: @niels9001